model/position.go

```diff
@@ -0,0 +1,185 @@
+package model
+
+import (
+	"time"
+)
+
+type MarginMode string
+
+const (
+	Isolated MarginMode = "ISOLATED"
+	Cross    MarginMode = "CROSS"
+
+	MAKER_FEE = 0.02
+	TAKER_FEE = 0.04
+)
+
+type Position struct {
+	ID            int64    `db:"id" json:"id" gorm:"primaryKey,autoIncrement"`
+	ExchangeID    int64    `db:"exchange_id" json:"exchange_id"`
+	Pair          string   `db:"pair" json:"pair"`
+	Side          SideType `db:"side" json:"side"`
+	EntryPrice    float64  `db:"entry_price" json:"entry_price"`
+	AvgPrice      float64  `db:"avg_price" json:"avg_price"`
+	Quantity      float64  `db:"quantity" json:"quantity"` //Asset e.g BTC
+	Size          float64  `db:"size" json:"size"`         // Quote e.g USDT
+	Active        bool     `db:"active" json:"active"`
+	Safe          bool     `db:"safe" json:"safe"`
+	Leverage      int
+	CurrentCandle *Candle
+	MarginMode    MarginMode `db:"margin_mode" json:"margin_mode"`
+
+	CreatedAt time.Time `db:"created_at" json:"created_at"`
+	UpdatedAt time.Time `db:"updated_at" json:"updated_at"`
+	ClosedAt  time.Time `db:"closed_at" json:"closed_at"`
+}
+
+func (p *Position) Closed() bool {
+	return p.ClosedAt.IsZero()
```

Comment:
I think that is the oposite sign? Correct?
```suggestion
	return !p.ClosedAt.IsZero()
```